features/dp-cantabular-csv-exporter-published-filtered.feature

```diff
@@ -0,0 +1,226 @@
+                                    "label": "5 siblings"
+                                },
+                                {
+                                    "code": "6",
+                                    "label": "6 or more siblings"
+                                }
+                            ],
+                            "count": 7,
+                            "variable": {
+                                "label": "Number of siblings",
+                                "name": "siblings"
+                            }
+                        }
+                    ],
+                    "error": null,
+                    "values": [
+                        1,
+                        0,
+                        0,
+                        1,
+                        0,
+                        0,
+                        0,
+                        0,
+                        0,
+                        0,
+                        0,
+                        1,
+                        0,
+                        0,
+                        0,
+                        0,
+                        1,
+                        0,
+                        0,
+                        1,
+                        1
+                    ]
+                }
+            }
+        }
+      }
+      """
+
+    And filter API is healthy
+    And dp-dataset-api is healthy
+    And cantabular server is healthy
+    And cantabular api extension is healthy
+
+    And the following instance with id "instance-happy-02" is available from dp-dataset-api:
+      """
+      {
+        "import_tasks": {
+          "build_hierarchies": null,
+          "build_search_indexes": null,
+          "import_observations": {
+            "total_inserted_observations": 0,
+            "state": "created"
+          }
+        },
+        "id": "057cd26b-e0ae-431f-9316-913db61cec39",
+        "last_updated": "2021-07-19T09:59:28.417Z",
+        "links": {
+          "dataset": {
+            "href": "http://localhost:22000/datasets/cantabular-dataset",
+            "id": "cantabular-dataset"
+          },
+          "job": {
+            "href": "http://localhost:21800/jobs/e7f99293-44f2-47ce-b6cb-db2f6618ef40",
+            "id": "e7f99293-44f2-47ce-b6cb-db2f6618ef40"
+          },
+          "self": {
+            "href": "http://10.201.4.160:10400/instances/057cd26b-e0ae-431f-9316-913db61cec39"
+          }
+        },
+        "state": "published",
+        "headers": [
+          "ftb_table",
+          "city",
+          "siblings"
+        ],
+        "is_based_on": {
+          "@type": "cantabular_table",
+          "@id": "Example"
+        }
+      }
+      """
+
+    And a dataset version with dataset-id "dataset-happy-02", edition "edition-happy-02" and version "version-happy-02" is updated by an API call to dp-dataset-api
+    And for the following filter "filter-happy-02" these dimensions are available:
+    """
+    {
+      "items": [
```

Comment:
You should be aware that these data models are going to change slightly very soon, you will probably have to come back and update these (and possibly in the code too)